modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -2021,6 +1882,128 @@ private ObjectDefinition _addUnmodifiableSystemObjectDefinition(
 				).build()));
 	}
 
+	private void _assertBoundedObjectDefinitions(
```

Comment:
Awesome idea!

I just suggest to separate each assertType on it's own method.
For example:
```
    _assertDataBasetableBoundedObjectDefinitions
    _assertPublishFailureBoundedObjectDefinitions
    _assertResourcesBoundedObjectDefinitions
    _assertStatusBoundedObjectDefinitions
```